webplugin/js/app/media/media-service.js

```diff
@@ -10,23 +32,31 @@ Kommunicate.mediaService = {
     processVoiceInputClickedEvent: function () {
         kmWidgetEvents.eventTracking(eventMapping.onVoiceIconClick);
         if (!('webkitSpeechRecognition' in window)) {
-            alert('browser do not support speech recogization');
+            alert('browser do not support speech recognition');
         } else {
-            var recognition = new webkitSpeechRecognition();
+            //As of April 2023, works only in chrome, edge and safari(limited support)
+            var recognizingDone = false;
+            var lastListeningEventTime = null;
```

Comment:
this is declared locally here but used outside too

Reply:
yes this is explicitly used in the set interval function to get the last time for which user speak